VirtualAssistantBusinessLogic/VirtualAssistantBusinessLogic/KnowledgeGraph/KnowledgeGraph.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Linq;
+using VirtualAssistantBusinessLogic.SparQL;
+using System.Collections.Generic;
+
+namespace VirtualAssistantBusinessLogic.KnowledgeGraph
+{
+    /// <summary>
+    /// Class representing the knowledge graph, that uses a SparQL Connection to
+    /// find information from the knowledge graph
+    /// </summary>
+    public class KnowledgeGraph
+    {
+        public KnowledgeGraph(SparQLConnectionFactory sparQLConnectionFactory)
+        {
+            SparQLConnectionFactory = sparQLConnectionFactory;
+        }
+
+        private SparQLConnectionFactory SparQLConnectionFactory { get; set; }
```

Comment:
Maybe move this to between line 6 and 7?

Reply:
Den skal ligge i klassen :)

Reply:
Up i staten af klassen XD